ultimatejobweb/templates/available_jobs.html

```diff
@@ -37,6 +37,30 @@
 				<td><a href="{{ item.description_url }}">{{ item.job_title }}</a></td>
 			</tr>
 				{% endfor %}
+            </tr>
```

Comment:
What is this data doing here? 
It does not follow the same format that the database generated data response (above).

Reply:
I removed the static data that i put inside the table (that was in there just as an example, and not actual job offers). 
Now that we have the algorithm that shows us the real job offers from Facebook and RedHat- these jobs will be inside the table.

Reply:
Just now committed these changes to the PR